### ISSUE TITLE
[BUG] Import Samples does not Work Properly for Dataset Type Yolov5

### ISSUE THREAD

**Issue by Yuxuan1998** [voxel51/fiftyone-plugins#203]
### Describe the problem
When I use io plugin to import Yolov5 samples to FiftyOne app, only validation set was successfully loaded despite all train, val and test sets are correctly defined in `dataset.yaml`. I have tried on different Yolov5 datasets, and all datasets were not loaded properly.

Here is my `dataset.yaml`:
```
train: ./train/images
val: ./valid/images
test: ./test/images

nc: 8
names: ['Cipro 500', 'Ibuphil 600 mg', 'Ibuphil Cold 400-60', 'Xyzall 5mg', 'blue', 'pink', 'red', 'white']
```

Here is my dataset folder structure:
```
<dataset_dir>/
    dataset.yaml
    train/
            images/
            labels/
     valid/
            images/
            labels/
     test/
            images/
            labels/

```

### Code to reproduce issue
```
import fiftyone as fo
import webbrowser

# create empty dataset
dataset = fo.Dataset("test-dataset")

# to view dataset in web browser
session = fo.launch_app(dataset, port=3030, auto=False)
webbrowser.open("http://localhost:3030")
```

<img width="975" alt="Image" src="https://github.com/user-attachments/assets/f575993d-4188-4ac4-a97f-ae4fc252a698" />

### System information
OS Platform and Distribution: macOS-15.3.1-arm64-arm-64bit 24.3.0
Python version : Python 3.11.3
FiftyOne version: 1.1.0
